### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pnbrown**
Hi folks, I worked on updating the README as follows. Please let me know if the changes are welcome and if there are any modifications you suggest.

- Updated text for clarity and readability.
- Removed bare links
- Stylized proper nouns
- Added header per [MD Style Guide](https://google.github.io/styleguide/docguide/style.html#document-layout)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-4. Validate if the version and the info is populated correctly from the https://github.com/valkey-io/valkey-hashes/blob/main/README file.
-5. See if all the tests pass on your fork and the your private docker hub has been updated.
-6. Update the `dockerhub-description.md` with the updated tags in your private docker hub and the Docker files links.
-7. Publish a PR with these changes. For example: https://github.com/valkey-io/valkey-container/pull/8
-8. Once the PR is merged, Sit back, relax and enjoy looking at your creation getting published to the official Docker hub page.
+1. Validate that the metadata for the new version is updated  the [Valkey hashes file](https://github.com/valkey-io/valkey-hashes/blob/main/README). If it is not updated, please open an issue [here](https://github.com/valkey-io/valkey-hashes/issues)
+2. If a new major or minor version is released, please create a new directory in the `valkey-container` repo e.g. [`7.2`](https://github.com/valkey-io/valkey-container/tree/mainline/7.2).
```
Comment (roshkhatri) [inline README.md]: `For example` would seem better

File: README.md [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+3. Run the `update.sh` script locally which will update `versions.json`. It will also populate the Dockerfiles for the new versions in the respective directories. Running the `update.sh` file executes `versions.sh` which updates `versions.json` with the required metadata from the [Valkey hashes file](https://github.com/valkey-io/valkey-hashes/blob/main/README). Once `versions.json` is updated, `apply-templates.sh` is executed which updates Dockerfiles for all the versions directories in the repo e.g. [`7.2`](https://github.com/valkey-io/valkey-container/tree/mainline/7.2).
+4. Validate that the version and the info is populated correctly from the [Valkey hashes file](https://github.com/valkey-io/valkey-hashes/blob/main/README).
+5. Verify all the tests pass on your fork and that your private Docker Hub repository has been updated.
+6. Update the `dockerhub-description.md` with the updated tags in your private Docker Hub registry and the Dockerfile links.
+7. Publish a PR with these changes, e.g. [#8](https://github.com/valkey-io/valkey-container/pull/8)
```
Comment (roshkhatri) [inline README.md]: `For example` would seem better

**Review by roshkhatri — COMMENTED** [review]
Over all LGTM just minor change revert

**Comment by pnbrown** [comment]
Reverted all e.g. to For example:

**Review by roshkhatri — APPROVED** [review]

**Pull request merged**